torch/csrc/autograd/variable.h

```diff
@@ -142,9 +149,6 @@ struct TORCH_API Variable : public at::Tensor {
 
   // NOTE: Assignment operators to Tensor come for free from the constructors.
 
-  const at::Tensor& data() const noexcept;
-  at::Tensor& data() noexcept;
```

Comment:
This is OK for now, but we might need to add these back for BC.

Reply:
(I say it's OK for now because I approve of expunging such reference in PyTorch codebase)